autoPyTorch/api/base_task.py

```diff
@@ -408,7 +412,7 @@ def _create_dask_client(self) -> None:
             dask.distributed.LocalCluster(
                 n_workers=self.n_jobs,
                 processes=True,
-                threads_per_worker=1,
+                threads_per_worker=2,
```

Comment:
should we have this as a user-defined parameter?

Reply:
yes, we should have it.

Reply:
This is weird, I have it as 2 on my branch?